addons/im_livechat/static/src/new/core/livechat_service.js

```diff
@@ -61,16 +72,21 @@ export class LivechatService {
         });
         this.available = init.available_for_me ?? this.available;
         this.rule = init.rule;
+        this.rule.chatbot = this.rule.chatbot ? new Chatbot(this.rule.chatbot) : undefined;
```

Comment:
It'd best to let chatbot service make chatbot model, and keep a single reference in chatbot service.
This will mean chatbot service owns chatbot model.